### PR TITLE
Added some sharing buttons

### PR DESCRIPTION
**Pull request by dragonito**

### CODE REVIEW

File: src/Olry/MentalNoteBundle/Resources/views/Entry/line.html.twig [hunk 1]
```diff
@@ -8,6 +8,11 @@
         <div class="col-sm-12 col-md-8">
             {% if show_actions|default(false) %}
             <div class="pull-right">
+
+                <a class="twitter-share-button" href="https://twitter.com/intent/tweet?text={{ entry.url }}" target="_blank"><i class="fa fa-twitter-square" aria-hidden="true"></i></a>
+
```
Comment (tolry) [inline src/Olry/MentalNoteBundle/Resources/views/Entry/line.html.twig]: I would remove this empty line

File: src/Olry/MentalNoteBundle/Resources/views/Entry/line.html.twig [hunk 2]
```diff
@@ -8,6 +8,11 @@
         <div class="col-sm-12 col-md-8">
             {% if show_actions|default(false) %}
             <div class="pull-right">
+
+                <a class="twitter-share-button" href="https://twitter.com/intent/tweet?text={{ entry.url }}" target="_blank"><i class="fa fa-twitter-square" aria-hidden="true"></i></a>
```
Comment (tolry) [inline src/Olry/MentalNoteBundle/Resources/views/Entry/line.html.twig]: * that class is probably from twitter's example code, no necessary here, is it?
* title missing

File: src/Olry/MentalNoteBundle/Resources/views/Entry/line.html.twig [hunk 3]
```diff
@@ -8,6 +8,11 @@
...
+
+                <a href="https://www.facebook.com/sharer/sharer.php?u={{ entry.url }}" target="_blank"><i class="fa fa-facebook-square" aria-hidden="true"></i></a>&nbsp;&nbsp;
```
Comment (tolry) [inline src/Olry/MentalNoteBundle/Resources/views/Entry/line.html.twig]: * title missing

File: src/Olry/MentalNoteBundle/Resources/views/Entry/line.html.twig (hunk 2 above)
Comment (dragonito) [inline src/Olry/MentalNoteBundle/Resources/views/Entry/line.html.twig]: Copy paste ill remove it

**Comment by dragonito** [comment]
About titles, fb renders opengraph data from link so titles could be a double information. In twitter i think its good to add the title.

Whats about adding tags als hashtags?

**Comment by tolry** [comment]
@dragonito wrong titles :wink: I was talking about the title attribute in the a-tags. So you will get a tooltip when hovering over the link.

**Comment by dragonito** [comment]
So hope its ok know @tolry

**Comment by tolry** [comment]
@dragonito almost :smile: I was thinking about having something like "share this link on facebook|twitter" as title attribute. The entry's title doesn't help the user much with the exact nature of the button which is just an icon.

**Comment by tolry** [comment]
@dragonito thanks!

**Pull request merged**